Source/ASTextNode.mm

```diff
@@ -450,10 +450,10 @@ - (void)setAttributedText:(NSAttributedString *)attributedText
   // Since truncation text matches style of attributedText, invalidate it now.
   [self _invalidateTruncationText];
   
-  NSUInteger length = attributedText.length;
+  NSUInteger length = _attributedText.length;
```

Comment:
Interesting fix, this could be important in the case of cleansing attributed text...